salvo/api/docker_volume.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package salvo;
+
+// This message defines the properties for a given mount. It is used
+// to generate the dictionary specifying volumes for the Python Docker
+// SDK.  This message is not constructed by the user
+message VolumeProperties {
+  // Specified the destination mount path in the benchmark container
+  string bind = 1;
+
+  // Define whether the mount point is read-write or read-only
+  // eg: 'rw' or 'ro'
+  string mode = 2;
```

Comment:
Probably should be an enum, but don't feel super strong.

Reply:
Enum it is.